test/screenshot/drawer/DrawerTest.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import '../../../packages/drawer/index.scss';
+import './index.scss';
+
+import TopAppBar from '../../../packages/top-app-bar/index';
+import MaterialIcon from '../../../packages/material-icon/index';
+import Drawer, {
+  DrawerAppContent,
+  DrawerHeader,
+  DrawerSubtitle,
+  DrawerTitle,
+  DrawerContent,
+} from '../../../packages/drawer/index';
+import List from './List';
+
+class DrawerScreenshotTest extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: props.open, // eslint-disable-line react/prop-types
+    };
+  }
+
+  render() {
+    const {title, dismissible, modal} = this.props; // eslint-disable-line react/prop-types
+    return (
+      <div className='drawer-screenshot-test'>
+        <Drawer
+          open={this.state.open}
+          onClose={() => this.setState({open: false})}
+          dismissible={dismissible}
+          modal={modal}
+        >
+          <DrawerHeader>
+            <DrawerTitle>
+              Inbox
+            </DrawerTitle>
+            <DrawerSubtitle>
+              ralph@gmail.com
+            </DrawerSubtitle>
+          </DrawerHeader>
+
+          <DrawerContent>
+            <List />
+          </DrawerContent>
+        </Drawer>
+
+        <DrawerAppContent>
+          <TopAppBar
+            title={title}
+            navigationIcon={this.renderNavigationIcon()}
+          />
+          <div className='mdc-top-app-bar--fixed-adjust'>
+            {[0, 1, 2, 3, 4, 5].map(this.renderLoremIpsum)}
+          </div>
+        </DrawerAppContent>
+      </div>
+    );
+  }
+
+  renderNavigationIcon() {
+    if (this.props.hideNavigationIcon) return; // eslint-disable-line react/prop-types
+    return (
+      <MaterialIcon onClick={() => this.setState({open: !this.state.open})} icon='menu' />
```

Comment:
set `tabIndex={0}` so we can see if focus goes to the app

Reply:
In modal, it actually just goes to the body. We would need to set this up. How about I add an issue to address this?